data/synapse-data-db2/pom.xml

```diff
@@ -35,12 +52,12 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
-            <version>6.1.7.Final</version>
+            <version>${hibernate.core.version}</version>
```

Comment:
emove version, this should already be specify in the parent pom

Reply:
done